docs/source/format/FlightSql.rst

```diff
@@ -170,6 +170,47 @@ the ``type`` should be ``ClosePreparedStatement``).
     When used with DoPut: execute the query and return the number of
     affected rows.
 
+Flight Server Session Management
+--------------------------------
+
+Flight SQL provides commands to set and update server session variables
+which affect the server behaviour in various ways.  Common options may
+include (depending on the server implementation) ``catalog`` and
+``schema``, indicating the currently-selected catalog and schema for
+queries to be run against.
+
+Clients should prefer, where possible, setting options prior to issuing
+queries and other commands, as some server implementations may require
+these options be set exactly once and prior to any other activity which
+may trigger their implicit setting.
+
+For compatibility with Database Connectivity drivers (JDBC, ODBC, and
+others), it is strongly recommended that server implementations accept
+string representations of all option values which may be provided to the
+driver as part of a server connection string and passed through to the
+server without further conversion.  For ease of use it is also recommended
+to accept and convert other numeric types to the preferred type for an
+option value, however this is not required.
+
+Sessions are persisted between the client and server using an
+implementation-defined mechanism, which is typically RFC 6265 cookies.
```

Comment:
Do we want to recommend a cookie name for Flight sessions?

Reply:
I'd thought about it but it's opaque to the client so being prescriptive isn't of much value.  The middleware shipped with this uses a name that service implementers are free to use, but it doesn't much matter, and I certainly don't want to imply that Flight cares.